spec/controllers/hyrax/collections_controller_spec.rb

```diff
@@ -8,7 +8,6 @@
     FactoryBot.valkyrie_create(:hyrax_collection,
                                :public,
                                title: ["My collection"],
-                               description: ["My incredibly detailed description of the collection"],
```

Comment:
The description attribute is not tested in any tests, so it was simply removed from the setup process.